### PR TITLE
Observe DOM changes in extensions

### PR DESCRIPTION
**Pull request by ghost**
Tip how to observe cascading DOM changes

### CODE REVIEW

**Comment by kurtextrem** [comment]
Some points for improvements:
- Add example usage (real function call)
- Add explanations for the parameters and or link to the MDN article

**Comment by homer0** [comment]
Besides @kurtextrem comments, 
- `extensions to existent sites`, are you talking about injecting/embedding content on a website?.
- [ ] whant?.
- [ ] Add more description, don't leave it just to the code to explain the tip.
- [ ] [squash your commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html).

**Comment by ghost** [comment]
Yes, I used this tip to build extension with injecting/adding functionality and content.

**Comment by kurtextrem** [comment]
Also useful for Chrome extensions (probably Firefox, Safari, Opera etc as well)

File: README.md [hunk 1]
```diff
@@ -17,6 +17,39 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter.
...
 # Tips list
+## # - Observe DOM changes in extensions
+
+> 2016-01- by [@beyondns](https://github.com/beyondns)
+
+When you develop extensions to existent sites it's not so easy to play with DOM
+'cause of modern dynamic javascript. [MutationObserver](https://developer.mozillaa.org/en/docs/Web/API/MutationObserver) is a solution to listen DOM changes and do what you whant to do with elements when they changed (added new ones, etc.). Follow this example:
+
+```javascript
+const observeConfig = {
```
Comment (loverajoel) [inline README.md:27]: you have strange indentation here

**Comment by sdgluck** [comment]
FYI the link to MDN `MutationObserver` documentation has the incorrect domain `mozillaa.org`.

**Comment by loverajoel** [comment]
Hi, how you can see the jstips repository changed his format and the `master` branch now is `gh-pages`, now are based in _jekyll blog format_. You can see live here => [jstips.com](http://www.jstips.co/)
For that reason you should update your tip with this [instructions](https://github.com/loverajoel/jstips/blob/gh-pages/CONTRIBUTING.md).

Let me know if you need some help or have suggestion. All are welcome!

Thanks

**Comment by ghost** [comment]
plain README.md was cool :)

**Comment by ghost** [comment]
I refactor my tip (working example now), do I need something more?

Thanks

**Comment by loverajoel** [comment]
@beyondns  hey use the PR template -> https://github.com/loverajoel/jstips/blob/gh-pages/GIT_TEMPLATE.md

**Comment by ghost** [comment]
Should I do new pull request to use new format?

**Comment by bhaskarmelkani** [comment]
Yes, you can fork a branch from `gh-pages` and after adding your changes you can create a PR with specified PR format.
Then once the PR is created, you can close this PR mentioning the PR id of the new one. :)

**Comment by loverajoel** [comment]
@beyondns ping!

**Comment by ghost** [comment]
pong https://github.com/loverajoel/jstips/pull/203

**Pull request closed without merging**